### ISSUE TITLE
Feature request: Integrate Telegram API Notifier Setup

### ISSUE THREAD

**Issue by ansred** [qdm12/gluetun#397]
**What's the feature?** 🧐
Integrate Telegram API Notifier Setup

- Support this new feature because that and that
This really good tool, when IP is being changed, port forwarded is being changed or any of your HTTP requests in the guide [https://github.com/qdm12/gluetun/wiki/HTTP-Control-server](url) I already use them via POSTMAN, but it would be great to use them with Telegram.

**Optional extra information** 🚀

- Examples: poma/docker-telegram-notifier - [https://github.com/poma/docker-telegram-notifier](url) is interesting, I have it setup with Tautulli, Ombi and other services
Here is their guide # https://github.com/Tautulli/Tautulli-Wiki/wiki/Notification-Agents-Guide#telegram

**Comment by kusold** [comment]
If you want to bake notifications into gluetun, you may want to give [shoutrrr](https://containrrr.dev/shoutrrr/) a look. It's a go library that supports sending to a wide variety of sources including telegram.

If you'd rather keep the notification service external, it would be useful to have etags or last-modified response headers so that a polling app can more easily see when the state changes.

**Comment by ansred** [comment]
I did a workaround in here:

https://github.com/qdm12/gluetun/discussions/446#discussioncomment-715376

**Comment by qdm12** [comment]
Yeah I'll use https://github.com/containrrr/shoutrrr/ in the end thanks @kusold
